src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizardStep2.java

```diff
@@ -118,30 +123,45 @@ public void configure(Table entity, TableType tableType) {
     this.entity = entity;
     this.tableType = tableType;
 
-    editor.configure(tableType, new ArrayList<ColumnModel>());
+    List<ColumnModel> defaultColumns = getDefaultColumns();
 
-    boolean isView = !TableType.table.equals(tableType);
-    this.editor.setAddDefaultColumnsButtonVisible(isView);
-    this.editor.setAddAnnotationColumnsButtonVisible(isView);
-    if (isView) {
-      // start with the default file columns
-      getDefaultColumnsForView();
+    if (DisplayUtils.isInTestWebsite(this.cookies)) {
+      editorV2.configure(
+        EntityTypeUtils.getEntityType(entity),
+        getViewScope(),
+        defaultColumns
+      );
+    } else {
+      editor.configure(tableType, new ArrayList<>());
+      boolean isView = !TableType.table.equals(tableType);
+      this.editor.setAddDefaultColumnsButtonVisible(isView);
+      this.editor.setAddAnnotationColumnsButtonVisible(isView);
```

Comment:
These visibility conditions are handled internally by the React component, so the methods on the widget are not 1:1